### ISSUE TITLE
The react-crud table doesn't come out expanded as seen in the code sandbox

### ISSUE THREAD

**Issue by snsarma** [xvicmanx/react-crud-table#3]
Hi 

The react-crud table doesn't get displayed as expanded as opposed to how it appears in code sandbox,
I need to click on the header in order to get it displayed, The code sand box has the table loaded or expanded but when I integrate the crud-table in my  code it appears collapsed and the column header needs to be clicked on for the data to b displayed , Could you please provide your inputs for this ?
Your help is much appreciated.

Thanks

**Comment by pawel54321** [comment]
Hi,

As for the content (data) of a given table, I had to do it like this:

`document.getElementsByClassName('crud-table__header-cell')[0].click(); `

In the sense, it automatically clicks the id (at the beginning ONLY once - bool).
I checked various methods and did not want to work. The example is just so that it "works" I do not know if it is correct.
I think that at least "something" will help

Best regards

**Comment by pawel54321** [comment]
Hi,

As for the content (data) of a given table, I had to do it like this:

`document.getElementsByClassName('crud-table__header-cell')[0].click(); `

In the sense, it automatically clicks the id (at the beginning ONLY once - bool, example trigger button/tabs).
I checked various methods and did not want to work. The example is just so that it "works" I do not know if it is correct.
I think that at least "something" will help.

Best regards

**Comment by pawel54321** [comment]
Hi,

As for the content (data) of a given table, I had to do it like this:

`document.getElementsByClassName('crud-table__header-cell')[0].click(); `

In the sense, it automatically clicks the id (example trigger: button/tabs).
I checked various methods and did not want to work. The example is just so that it "works" I do not know if it is correct.
I think that at least "something" will help.

Best regards
